### PR TITLE
[Android] Remove ad-hoc distribution of keyboards via keyman:// protocol

### PR DESCRIPTION
**Pull request by darcywong00**
Addresses Android platform for #538

### CODE REVIEW

File: android/history.md [hunk 1]
```diff
@@ -3,7 +3,8 @@
 
 ## 11.0 alpha
 * Move to 11.0
-* Add round launcher icons
+* Add round launcher icons (#1077)
+* Breaking change - Remove ad-hoc distribution of keyboards via `keyman://` protocol (#1109)
```
Comment (mcdurdin) [inline android/history.md]: Suggest we don't use phrase "Breaking change" because while it is a breaking change, it's more of a removal of old infrastructure. "Breaking change" sounds scary but this isn't really scary -- vast majority of people won't even notice. Instead, maybe `Remove deprecated ad-hoc distribution of keyboards via `keyman://` protocol (#1109)`

File: android/KMEA/app/src/main/java/com/tavultesoft/kmea/KMKeyboardDownloaderActivity.java [hunk 2]
```diff
@@ -253,48 +257,23 @@ protected int downloadNonKMPKeyboard(String remoteUrl) throws Exception {
...
-          throw new Exception(exceptionStr);
-        }
-      } else {
-        // Legacy method of ad-hoc keyboard distribution via JSON
-        keyboard = kbData.optJSONObject(KMKey_Keyboard);
-        if (keyboard == null) {
-          throw new Exception(exceptionStr);
-        }
-
-        JSONArray languages = keyboard.optJSONArray(KMKey_Languages);
```
Comment (mcdurdin) [inline android/KMEA/app/src/main/java/com/tavultesoft/kmea/KMKeyboardDownloaderActivity.java]: Are some of the `KMKey_` constants now unused?

File: android/KMEA/app/src/main/java/com/tavultesoft/kmea/KMKeyboardDownloaderActivity.java [hunk 3]
```diff
@@ -226,8 +226,7 @@ protected void onPostExecute(Integer result) {
     }
 
     /**
-     * Download a non-KMP Keyman keyboard.
-     * This will either be from Keyman cloud or legacy ad-hoc distribution via JSON
+     * Download a non-KMP Keyman keyboard from Keyman cloud via JSON
```
Comment (mcdurdin) [inline android/KMEA/app/src/main/java/com/tavultesoft/kmea/KMKeyboardDownloaderActivity.java]: We'll be removing code for pulling non-KMP downloads from Keyman cloud as well soon -- i.e. downloading all files by KMP from Keyman cloud as well. But can put that in a separate PR once we are sure the backend is ready for it.

Comment (darcywong00) [reply android/KMEA/app/src/main/java/com/tavultesoft/kmea/KMKeyboardDownloaderActivity.java]: 👍 btw, the diff in this file is cleaner when ignoring whitespace
```
https://github.com/keymanapp/keyman/pull/1109/files?w=1
```
